queue_services/entity-filer/src/entity_filer/filing_processors/registration.py

```diff
@@ -116,25 +112,24 @@ def process(
     if not (firm_reg_num := legal_entity_info.get_next_corp_num("FM")):
         raise DefaultException(f"registration {filing_rec.id} unable to get a Firm registration number.")
 
-    match legal_type:
-        case LegalEntity.EntityTypes.SOLE_PROP:
+    match entity_type:
+        case BusinessCommon.EntityTypes.SOLE_PROP:
             # Get or create LE
-            business = merge_sp_registration(firm_reg_num, filing, filing_rec)
+            business, alternate_name = merge_sp_registration(firm_reg_num, filing, filing_rec)
 
-        case LegalEntity.EntityTypes.PARTNERSHIP:
+        case BusinessCommon.EntityTypes.PARTNERSHIP:
             # Create Partnership
-            business = merge_partnership_registration(firm_reg_num, filing, filing_rec, registration_filing)
+            business, alternate_name = merge_partnership_registration(
+                firm_reg_num, filing, filing_rec, registration_filing
+            )
 
         case _:
             # Default and failed
             # Based on the above checks, this should never happen
             raise DefaultException(f"registration {filing_rec.id} had no valid Firm type.")
 
-    # Assuming we should not reset this from a filing
-    if not business.tax_id:
-        business.tax_id = registration_filing.get("business", {}).get("taxId", None)
-
-    business.state = LegalEntity.State.ACTIVE
+    business.state = BusinessCommon.State.ACTIVE
```

Comment:
so `business` here can potentially be colin entity or a LE person record right?  

Don't think we need to set the state for such scenarios.  Only the alternate name record will need the state set.  

And in the case of colin entity, I think it will error out.

Reply:
Yes it could be. Thanks for catching it! 😸 So only update SP & GP themselves right?

Reply:
Will remove this line and the line below since it's set `ACTIVE` by default.